libraries/hermes/src/Rest/Swagger.ts

```diff
@@ -131,6 +132,23 @@ export class SwaggerGenerator {
     routeDoc.responses[200].content['application/json'].schema = {
       $ref: `#/components/schemas/${route.returnTypeName}`,
     };
+    if (!isNil(routeDoc['requestBody']) || !isNil(routeDoc['parameters'])) {
+      routeDoc.responses[400] = {
+        description:
+          'Invalid parameters provided. Check the documentation for more info.',
+      };
+    }
+    if (route.input.middlewares?.includes('authMiddleware')) {
```

Comment:
This segment should ideally be abstracted away so that individual routers may provide provide their own options individually from `hermes`.

Reply:
This should also handle `authMiddleware?`